chapter1/src/main/java/spring/training/controller/MyFirstController.java

```diff
@@ -0,0 +1,42 @@
+package spring.training.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import spring.training.model.Name;
+
+@Controller
+public class MyFirstController {
+	private String message = "Hello ";
+	
+	public String getMessage() {
+		return message;
+	}
+
+	
+	public void setMessage( String message ) {
+		this.message = message;
+	}
+
+	
+	public Name getName() {
+		return name;
+	}
+
+	
+	public void setName( Name name ) {
+		this.name = name;
+	}
+
+	@Autowired
```

Comment:
May be i am wrong. But i have always end up problems when i autowire in sometime.